plugins/autoloader/prism-autoloader.js

```diff
@@ -182,8 +182,7 @@
 
 	var ignored_language = 'none';
 
-	var scripts = document.getElementsByTagName('script');
-	var script = scripts[scripts.length - 1];
+	var script = Prism.util.currentScript();
```

Comment:
We need to check `script` exists in the usages below.

Reply:
Is there a case where it doesn't exist?

Reply:
Inline scripts? We check if `script` exists first in our other usages as well.

Reply:
Well, true. My bad.

Reply:
Maybe I've been working in TS for too long, but my thought is just that we know this function can return `null` so we should check before we interact with it.

Reply:
Nah, it's alright.
Actually, my first through when changing this to `Prism.util.currentScript()` was: "TS won't approve of this".